### ISSUE TITLE
Red-Science trains useless without train stops & signals

### ISSUE THREAD

**Issue by SWeini** [pyanodon/pycoalprocessing#254]
I could understand if fluid wagons are unlocked with green science. Long-distance pipes are not extremely expensive.
But trains with only red science are useless if you can't build stations or signals.

<img width="255" alt="image" src="https://user-images.githubusercontent.com/22458042/150562796-67562173-aeda-42bf-9e7b-7ffbdb8ef0a5.png">

**Comment by dengamu** [comment]
I would disagree that trains are useless if you cannot build stations or signals. You would be unable to automate trains without at least stations yes.  Signals would not be required for automation, especially if there is only one train.  It would just be a bit more difficult to path rails for no intersections.  However, for the main issue, trains could be used solely for manual driving without stations or signals being required.  Most notably, for travel between commonly visited points, simply laying rail and using a locomotive with/without wagons can greatly increase travel speed.
